source/core/src/main/com/csse3200/game/entities/configs/WeaponConfig.java

```diff
@@ -15,8 +15,8 @@ public class WeaponConfig {
 
     static {
         // Load weapon data (can be from JSON or hardcoded for now)
-        weaponConfigs.put("Shotgun", new WeaponData("Shotgun", 10, 10,
-                5, 20, 20, 3, "images" +
+        weaponConfigs.put("Shotgun", new WeaponData("Shotgun", 15, 3,
```

Comment:
The Functioning is correct, better to do it as a JSON, we might be able to change it based on difficulty level